app/models/business.rb

```diff
@@ -12,6 +12,7 @@ class Business < ApplicationRecord
   validates :company, presence: true
   validates :bio, presence: true
   validates :developer_notifications, inclusion: {in: developer_notifications.keys}
+  validates :website_url, url: true
```

Comment:
If we are using the gem I think we also need to allow blanks.

```suggestion
  validates :website_url, url: {allow_blank: true}
```